zeebe/engine/src/main/java/io/camunda/zeebe/engine/scaling/ScaleUpProcessor.java

```diff
@@ -54,6 +54,13 @@ public void processRecord(final TypedRecord<ScaleRecord> command) {
       return;
     }
 
+    if (scaleUp.desiredPartitionCount() < 1 || scaleUp.currentPartitionCount() < 1) {
```

Comment:
Well not really, the user will not be able to provide this value, it will be generated internally instead. We _could_ try and figure it out based on the current routing state but that is a bit more error prone(eventual consistency etc.)